policy/modules/services/console_setup.if

```diff
@@ -0,0 +1,106 @@
+## <summary>console font and keymap setup program for debian</summary>
+
+########################################
+## <summary>
+##  Execute console-setup in the console_setup domain.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed to transition.
+##	</summary>
+## </param>
+#
+interface(`console_setup_domtrans', `
+    gen_require(`
+        type console_setup_t, console_setup_exec_t;
+        type console_setup_conf_t;
+        type console_setup_runtime_t;
+    ')
+
+    corecmd_search_bin($1)
+    domtrans_pattern($1, console_setup_exec_t, console_setup_t)
+')
+
+########################################
+## <summary>
+##  Read console-setup configuration files.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+## <rolecap/>
+#
+interface(`console_setup_read_config',`
+	gen_require(`
+        type console_setup_conf_t;
+	')
+
+    files_search_etc($1)
+    allow $1 console_setup_conf_t:dir list_dir_perms;
+    allow $1 console_setup_conf_t:file read_file_perms;
+    allow $1 console_setup_conf_t:lnk_file read_lnk_file_perms;
+')
+
+########################################
+## <summary>
+##  Execute console-setup configuration files
+##  in the caller domain.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed to transition.
+##	</summary>
+## </param>
+#
+interface(`console_setup_execute_conf', `
```

Comment:
"exec" not "execute". Also be consistent and use either "conf" or "config" for this and the above interface name.